code/datums/mutations/body.dm

```diff
@@ -430,3 +430,65 @@
 			owner.SetStun(owner.AmountStun()*2)
 			owner.visible_message(span_danger("[owner] tries to stand up, but trips!"), span_userdanger("You trip over your own feet!"))
 			stun_cooldown = world.time + 300
+
+/datum/mutation/human/hypermarrow
+	name = "Hyperactive Bone Marrow"
+	desc = "A mutation that stimulates the subject's bone marrow causes it to work three times faster than usual."
+	quality = POSITIVE
+	text_gain_indication = span_notice("You feel your bones ache for a moment.")
+	text_lose_indication = span_notice("You feel something in your bones calm down.")
+	difficulty = 12
+	instability = 20
+	power_coeff = 1
+
+/datum/mutation/human/hypermarrow/on_life()
+	if(owner.blood_volume < BLOOD_VOLUME_NORMAL(owner))
+		owner.blood_volume += GET_MUTATION_POWER(src) * 2 - 1
+		owner.physiology.hunger_mod *= GET_MUTATION_POWER(src) * 2 - 0.8
```

Comment:
```suggestion
		owner?.physiology?.hunger_mod *= GET_MUTATION_POWER(src) * 2 - 0.8
```

Reply:
I don't really understand what the question mark does but it and the rest of the suggestions have been committed.

Reply:
> I don't really understand what the question mark does but it and the rest of the suggestions have been committed.

it checks if the var is null so it doesn't runtime, works the same as doing if(owner && physiology && ...)